shared/src/main/scala/squants/Quantity.scala

```diff
@@ -123,7 +123,7 @@ abstract class Quantity[A <: Quantity[A]] extends Serializable with Ordered[A] {
    * @return Quantity
    */
   def negate: A = unit(-value)
-  def unary_-(): A = negate
+  def unary_- : A = negate
```

Comment:
This looks like it should be a binary-compatible change. If `unary_-` had an empty parameter lists, Dotty would prevent you from calling it as `-x`.

Reply:
I was planning to put this as a separate PR to be included in the current master. This triggers warnings on 2.13.3